ngraph/test/CMakeLists.txt

```diff
@@ -105,6 +105,7 @@ set(SRC
     shape.cpp
     specialize_function.cpp
     tensor.cpp
+    type_prop/arithmetic_ops.hpp
```

Comment:
This is list of sources, we shouldn't add headers here. I think you don't need to add it to cmake at all, compiler should find it since it's in the same directory as source file.